packages/adapter-netlify/index.js

```diff
@@ -126,10 +126,42 @@ async function generate_edge_functions({ builder }) {
 
 	builder.mkdirp('.netlify/edge-functions');
 
+	builder.log.minor('Generating Edge Function...');
+	const relativePath = posix.relative(tmp, builder.getServerDirectory());
+
+	builder.copy(`${files}/edge.js`, `${tmp}/entry.js`, {
+		replace: {
+			'0SERVER': `${relativePath}/index.js`,
+			MANIFEST: './manifest.js'
+		}
+	});
+
+	const manifest = builder.generateManifest({
+		relativePath
+	});
+
+	writeFileSync(`${tmp}/manifest.js`, `export const manifest = ${manifest};\n`);
+
+	/** @type {{ assets: Set<string> }} */
+	const { assets } = (await import(`${tmp}/manifest.js`)).manifest;
+
 	const path = '/*';
+	// We only need to specify paths without the trailing slash because
+	// Netlify will handle the optional trailing slash for us
 	const excludedPath = [
 		// Contains static files
 		`/${builder.getAppPath()}/*`,
+		...builder.prerendered.paths,
+		...Array.from(assets).flatMap((asset) => {
```

Comment:
the previous logic also checked within `manifest._.server_assets`, this one does not - oversight or on purpose?

Reply:
We could add `manifest._.server_assets` too but they're currently included in the immutable assets path too `/${builder.getAppPath()}/*` because we copy the server assets there and make them public. I'm not sure if that will change in the future.